help/sources/connectors/data-partners/algolia-user-profiles.md

```diff
@@ -0,0 +1,21 @@
+# [!DNL Algolia User Profiles]
+
+## Introduction
+[!DNL Adobe Experience Platform] (AEP) RealTime Customer Data Platform (CDP) integrates multiple user management platforms to collect disparate user profiles to build holistic profiles for personalization and experience testing. The platform provides out of the box integrations to pull user profiles from source systems. It also provides a SDK to build custom integrations. [!DNL Algolia] has created a Source Connector that integrates with Algolia’s Advanced Personalization APIs to retrieve user profiles into AEP.  This document will guide you through the set-up of the [!DNL Algolia User Profiles] Source connector.
```

Comment:
- We can remove the heading for "Introduction" as the H1 header should be enough as primary intro.
- Let's remove acronyms as well, so no "AEP"
- However, let's update "RealTime Customer Data Platform (CDP)" -> "Real-Time CDP".
- "The platform provides out of the box integrations to pull user profiles from source systems." -> "Experience Platform provides integrations to pull user profiles from a variety of sources systems."
- "[!DNL Algolia] has created a Source Connector that integrates with Algolia’s Advanced Personalization APIs to retrieve user profiles into AEP." -> "Use the [!DNL Algolia] source to integrate with [!DNL Algolia's] Advanced Personalization APIs and retrieve user profiles for ingestion into Experience Platform."

Reply:
ok, i have applied the changes.  thanks